shared/src/helpers/configure-date/configure-date.ts

```diff
@@ -0,0 +1,16 @@
+const configureDate = (
```

Comment:
We don't have similar helper? And as I can see you are not using it on backend, so maybe worth to move it to frontend and utilize date-fns to simplify